real_time/surface_src/java_src/Alice/src/test/java/TestSo2Pose.java

```diff
@@ -1,105 +1,109 @@
-import static org.junit.Assert.*;
-
-import org.junit.After;
-import org.junit.Before;
+import com.roboclub.robobuggy.map.So2Pose;
 import org.junit.Test;
 
-import com.roboclub.robobuggy.main.Util;
-import com.roboclub.robobuggy.map.So2Pose;
+import static org.junit.Assert.fail;
 
 
+/**
+ * Tests aspects of the So2Pose
+ */
 public class TestSo2Pose {
 
-	@Before
-	public void setUp() throws Exception {
-	}
-
-	@After
-	public void tearDown() throws Exception {
-	}
 
+	/**
+	 * basic test of the so2pose
+	 */
 	@Test
 	public void test() {
-		So2Pose A = new So2Pose(0.0, 0.0, 0.0);
-		So2Pose B = new So2Pose(0.0, 1.0, 0.0);
-		So2Pose C = A.mult(B);
-		if(C.getY() != 1.0){		
+		So2Pose a = new So2Pose(0.0, 0.0, 0.0);
+		So2Pose b = new So2Pose(0.0, 1.0, 0.0);
+		So2Pose c = a.mult(b);
+		if(c.getY() != 1.0){
 			fail("y value did not become 1.0 when it should ");
 		}		
 		
 		//checking the inverse 
-		C = A.mult(B.inverse());
-		if(C.getY() != -1.0){		
+		c = a.mult(b.inverse());
+		if(c.getY() != -1.0){
 			fail("y value did not become -1.0 when it should ");
 		}
 		
 	}
-	
+
+	/**
+	 * tests whether 2 so2poses are equal
+	 */
 	@Test
-	public void test_equal(){
-		So2Pose A = new So2Pose(1.0,2.0,3.0);
-		So2Pose B = new So2Pose(1.0,2.0,3.0);
-		if(!A.equals(B)){
+	public void testEqual(){
+		So2Pose a = new So2Pose(1.0,2.0,3.0);
+		So2Pose b = new So2Pose(1.0,2.0,3.0);
+		if(!a.equals(b)){
 			fail("So2Equal was not written correctly ");
 		}
 		
-		if(!B.equals(A)){
+		if(!b.equals(a)){
 			fail("So2Equal was not written correctly ");
 		}
 		
-		So2Pose C = new So2Pose(1.1,2.0,3.0);
-		if(C.equals(A)){
+		So2Pose c = new So2Pose(1.1,2.0,3.0);
+		if(c.equals(a)){
 			fail("So2Equal was true when it should not be");
 		}
 	}
-	
-	@Test 
-	public void test_inverse(){
-		So2Pose A = new So2Pose(1.0,2.0,3.0);
-		So2Pose B = A.inverse();
 
-		So2Pose C = B.mult(A);
-		if(!C.equals(So2Pose.identity())){
+	/**
+	 * tests whether the inverse works
+	 */
+	@Test
+	public void testInverse(){
+		So2Pose a = new So2Pose(1.0,2.0,3.0);
+		So2Pose b = a.inverse();
+
+		So2Pose c = b.mult(a);
+		if(!c.equals(So2Pose.identity())){
 			fail("matrix inverse time original is not identity");
 		}
 		
 		
-		So2Pose D = A.mult(B);
-		if(!D.equals(So2Pose.identity())){
+		So2Pose d = a.mult(b);
+		if(!d.equals(So2Pose.identity())){
 			fail("matrix inverse time original is not identity");
 		}
 	}
-	
+
+	/**
+	 * tests basic compounding of matrices
+	 */
 	@Test
-	public void test_2(){
+	public void test2(){
 		
 		// test basic compounding of matrices
-		So2Pose A = new So2Pose(0.0,1.0,0.0);
-		So2Pose B = new So2Pose(0.0,2.0,0.0);
-		So2Pose C = A.mult(B);
+		So2Pose a = new So2Pose(0.0,1.0,0.0);
+		So2Pose b = new So2Pose(0.0,2.0,0.0);
+		So2Pose c = a.mult(b);
 		
-		if(C.getY() != 3.0){
+		if(c.getY() != 3.0){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2369456130/issues/source?bid=2938665&fileBranchId=3216826#l85)